app/main.py

```diff
@@ -1,6 +1,30 @@
+import os
+
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    data = {
+        "key": os.environ["API_KEY"],
+        "q": "Paris",
+    }
```

Comment:
Read the checklist, you should add constants:

![image](https://user-images.githubusercontent.com/104218550/211518357-d402a74a-00ed-433c-9a66-1f1197b57d44.png)